src/main/resources/chains/etc-chain.conf

```diff
@@ -0,0 +1 @@
+{ include "base-chain.conf"}
```

Comment:
Do we need `base-chain`?

Reply:
It was created as etc and eth networks share a lot of common configuration

Reply:
Yes, but base == etc. In my opinion, we don't need it

Reply:
Hmm I'm also a bit on Michal's side in can be a bit confusing

In my opinion I'd just replicate the duplicated configuration in both etc-chain.conf and eth-chain.conf, just to have to check a single file to see all the configurations of a chain. This kinds of files will rarely be touched so I don't see that being too much of a problem

Reply:
Go for it then!

Reply:
Just updated them! Please check that I haven't screwed up in the middle

I noticed some ropsten configuration wasn't properly configured but I'd leave it as is to not fall down in that rabbit hole, if we ever start providing official support for it (seems to be the intent eventually) we'll fix it then

Reply:
I prefer to stay with `etc.conf` instead of `base.conf`. From a user perspective `mantis-launcher etc` makes more sense than `mantis-launcher base`